pyproject.toml

```diff
@@ -71,7 +71,7 @@ tests = [
     "pytest-rerunfailures==12.0",
 ]
 doc = [
-    "sphinx==6.2.1",
+    "sphinx==7.0.1",
```

Comment:
We cannot update to sphinx 7 yet:

https://github.com/ansys/ansys-sphinx-theme/blob/4c8a980ec82f0e54459c8f324863dbf1ae84b0a0/pyproject.toml#L36